openedx/features/partners/tests/test_login.py

```diff
@@ -0,0 +1,96 @@
+from django.urls import reverse
+from openedx.features.partners.tests.factories import PartnerUserFactory, PartnerFactory
+from rest_framework.test import APITestCase
+from rest_framework import status
+from student.tests.factories import UserFactory
+
+
+class ResetPasswordTestCases(APITestCase):
+
+    def setUp(self):
+        self.end_point = reverse('partner_login',args=['give2asia'])
```

Comment:
Lets rename `end_point`, which reflects what end point it is

Reply:
Done.